src/osmf/js/osmf_flash.js

```diff
@@ -774,13 +787,31 @@ require("../../../html5-common/js/utils/constants.js");
         break;
       }
       return true;
-    }, this);
+    }
+
   };
+
   /************************************************************************************/
   // Helper methods
   /************************************************************************************/
 
   /**
+   * Returns the SWF instance present in the Dom matching the provided id
+   * @private
+   * @method OoyalaFlashVideoWrapper#getSwf
+   * @param {string} thisId the id of the swf object sought.
+   * @returns {object} the object containing the desired swf.
+   */
+  var getSwf = function (thisId) {
+    if (navigator.appName.indexOf("Microsoft") != -1) {
+      return document.getElementsByName(thisId)[0];
+    }
+    else{
+      return document.getElementsByName(thisId)[0];
+    }
+  };
```

Comment:
Accidental holdover from prior code, where this function was elsewhere and there was a difference (Microsoft check originally targeted to 'window'). Removed that if / else.